java-checks/src/test/files/checks/LeastSpecificTypeCheck.java

```diff
@@ -153,7 +153,7 @@ public static void visibility(Visibility vis) { // Compliant - Base has package
 
   protected static class Visibility2 extends ProtectedBase { }
 
-  public static void visibility(Visibility2 vis) { // // Noncompliant {{Use 'A.ProtectedBase' here; it is a more general type than 'Visibility2'.}}
```

Comment:
This one is interesting... it never worked. unifying the verifiers made that comment invalid, and I realized we were not raising issue here. I didn't investigated if it's expected or not.